packages/ember-application/lib/system/application.js

```diff
@@ -352,8 +356,13 @@ const Application = Engine.extend({
     this._booted = false;
 
     if (isEnabled('ember-application-visit')) {
-      this.autoboot = this._globalsMode = !!this.autoboot;
+      if (typeof document === 'undefined' && typeof process !== 'undefined') {
+        this.autoboot = this._globalsMode = false;
+      }
 
+      // can we remove these code paths here if we're in node after the feature
+      // flag is removed? autoboot: true always blows up in node anyway, do we
+      // have a use case within node?
       if (this._globalsMode) {
         this._prepareForGlobalsMode();
```

Comment:
no globals in node, so superfluous

(learning still so thanks for the feedback where im wrong)